x-pack/plugins/osquery/server/routes/fleet_wrapper/get_osquery_available.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
can this be just exposed as a plugin method, not as a rest API?

Reply:
removed the api. Now I am making api calls from within the component as you suggested. 👍